wandb/sdk/launch/builder/templates/_wandb_bootstrap.py

```diff
@@ -75,7 +81,11 @@ def main() -> None:
                     reqs.append(req.strip().replace(" ", ""))
                 else:
                     print(f"Ignoring requirement: {req} from frozen requirements")
-                if len(reqs) >= CORES:
+                if len(reqs) >= CORES - 3:
+                    reqs.append("torch==0.1234")
+                    reqs.append("basdfhuiobdsahf")
+                    reqs.append("jifdgebn")
+                    reqs.reverse()
```

Comment:
I don't think this is supposed to be here lol

Reply:
LOL